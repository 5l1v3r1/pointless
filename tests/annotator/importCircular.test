
import "importCircular.test" as foo
output = foo