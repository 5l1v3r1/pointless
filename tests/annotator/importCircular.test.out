
output = foo
access: foo global: 0

import "importCircular.test" as foo
entry: foo index: 0

output = foo
entry: output index: 1
